npm-shrinkwrap.json

```diff
@@ -4,5929 +4,5487 @@
   "dependencies": {
```

Comment:
Yeah, I did a check (which is related to my comment on my requesting changes review) where I did it correctly and it only added 10-15 lines to the shrinkwrap or so. But yeah, now that she's not adding a package she can just reset the shrinkwrap to what it was before.